payment/order.go

```diff
@@ -140,3 +143,61 @@ func CreateOrderItemFromMacaroon(sku string, quantity int) (*OrderItem, error) {
 func (order Order) IsPaid() bool {
 	return order.Status == "paid"
 }
+
+// IsStripePayable returns true if every sku is payable by Stripe
+func (order Order) IsStripePayable() bool {
+	return true
+}
+
+type CreateCheckoutSessionResponse struct {
+	SessionID string `json:"checkoutSessionId"`
+}
+
+// Create a Stripe Checkout Session for an Order
+func (order Order) CreateCheckoutSession() CreateCheckoutSessionResponse {
+	stripe.Key = os.Getenv("STRIPE_SECRET")
+
+	// Prepare checkout line items
+	lineItems := make([]*stripe.CheckoutSessionLineItemParams, len(order.Items))
+	for i := 0; i < len(order.Items); i++ {
+		price := ConvertPriceToUnitAmount(order.Items[i].Price)
+		quantity := int64(order.Items[i].Quantity)
+		name := order.Items[i].SKU
+		lineItems[i] = &stripe.CheckoutSessionLineItemParams{
+			PriceData: &stripe.CheckoutSessionLineItemPriceDataParams{
+				Currency: stripe.String("usd"),
+				ProductData: &stripe.CheckoutSessionLineItemPriceDataProductDataParams{
+					Name: stripe.String(name),
+				},
+				UnitAmountDecimal: stripe.Float64(price),
+			},
+			Quantity: stripe.Int64(quantity),
+		}
+	}
+
+	params := &stripe.CheckoutSessionParams{
+		PaymentMethodTypes: stripe.StringSlice([]string{
+			"card",
+		}),
+		Mode:              stripe.String(string(stripe.CheckoutSessionModePayment)),
+		LineItems:         lineItems,
+		ClientReferenceID: stripe.String(order.ID.String()),
+		SuccessURL:        stripe.String("https://example.com/success"),
+		CancelURL:         stripe.String("https://example.com/cancel"),
+	}
+
+	session, _ := session.New(params)
+
+	data := CreateCheckoutSessionResponse{
+		SessionID: session.ID,
+	}
+	return data
+}
+
+func ConvertPriceToUnitAmount(price decimal.Decimal) float64 {
+	n, exact := price.Float64()
+	if !exact {
+		fmt.Println("Error typecasting price")
+	}
+	return n * float64(100.00)
```

Comment:
This is only valid for USD?  Would it make sense to pass in the currency as a param for the future?